src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -539,7 +539,7 @@ private static bool DirectoryContainsFullBackupOrSnapshot(string fullPath, Backu
             where S : BackupSettings
             where T : CloudUploadStatus
         {
-            if (CanBackupUsing(settings) == false)
+            if (PeriodicBackupConfiguration.CanBackupUsing(settings) == false)
```

Comment:
If we refuse to backup something that was configured, we need to add an operations log to that.